bigdata/src/main/scala/org/w3/banana/bigdata/package.scala

```diff
@@ -0,0 +1,48 @@
+package org.w3.banana
+
+import org.openrdf.model.{URI, Resource, Statement, Value}
+import org.openrdf.query.{GraphQueryResult, BindingSet, TupleQueryResult}
+import org.openrdf.repository.RepositoryResult
+import scala.collection.immutable.{Map, List}
+
+/**
+ * I put here some implicits to add Iterable support for bigdata StatementIterators
+ * as they do not implement Java iterator/iterable so JavaConversions do not work for them
+ */
+package object bigdata extends BigdataResultsConversions
+
+trait BigdataResultsConversions
```

Comment:
Let's not shadow the implicits in the main package. Let's have them in an object, and the code must import them explicitly.